lib/analysis_options.0.0.1.yaml

```diff
@@ -97,9 +97,13 @@ dart_code_metrics:
     - avoid-unrelated-type-assertions
     - avoid-unused-parameters
     - ban-name:
+        exclude:
+          - domain/lib/time/time_provider.dart
         entries:
           - ident: DateTime.now
             description: Please use TimeProvider.now() instead
+          - ident: SharedPreferences.getInstance
+            description: Please use stats repository instead
```

Comment:
The stats repo is specific to your project, this should probably be "designated"